src/main/scala/com/fulcrumgenomics/bam/SortBam.scala

```diff
@@ -41,6 +41,10 @@ import com.fulcrumgenomics.util.Io
     |and several
     |4. **RandomQuery**: sorts the reads into a random order but keeps reads with the same
     |   queryname together. The ordering is deterministic for any given input.
+    |5. **TemplateCoordinate**: sorts the reads The sort order used by `GroupReadByUmi`. Sorts reads by
```

Comment:
```suggestion
    |5. **TemplateCoordinate**: The sort order used by `GroupReadByUmi`. Sorts reads by
```

I think this was a copy/paste artifact from the preceding line